test/cmocka/src/common_mocks.c

```diff
@@ -97,6 +97,7 @@ int WEAK rstrlen(const char *s)
 void WEAK __panic(uint32_t p, char *filename, uint32_t linenum)
 {
 	fail_msg("panic: %s:%d (code 0x%x)\n", filename, linenum, p);
+	exit(1);
```

Comment:
I think I pushed wrong patch version, this should be `exit(EXIT_FAILURE);`

Reply:
To be fair they're equivalent at runtime (EXIT_FAILURE is 1 I think)

Reply:
I just fixed it.